### ISSUE TITLE
STS acronym

### ISSUE THREAD

**Issue by peppelinux** [vcstuff/draft-looker-jwt-cwt-status-list#9]
In the OAuth2 ecosystem we have already the acronym STS, as Security Token Service, as defined in Token Exchange RFC 8693

why not simply `status`?

**Comment by tplooker** [comment]
Thanks for flagging, yes we can consider this,  I was trying to align to the informal convention JWT's typically use for root level claims which are strings with 3 characters.

**Comment by peppelinux** [comment]
I love this sensibility, even if from years at this part this convention is not respected anymore

`sta` is weird
`stat` is something good even if outside the three chars, at the same time it may be misleading since stat may point also to statistics

who knows if `status` besides being explicit represents the best compromise after all

**Comment by tplooker** [comment]
Agreed, status feels like the most sensible compromise, I will prepare a PR that makes this change.

**Comment by paulbastian** [comment]
Agreed, please also adopt this for the status_list

**Comment by paulbastian** [comment]
Close with #16 ?
